tests/functional/graphql/test_related.py

```diff
@@ -157,6 +158,8 @@ def after_save_json(item_id: str):
 qdrant_error_mock = mock.Mock()
 qdrant_error_mock.side_effect = UnexpectedResponse(500, 'error', None, None)
 
+logging.getLogger().setLevel(logging.INFO)
```

Comment:
Suggestion: Assert that the log statement matches the expected format. For example: https://github.com/Pocket/recommendation-api/blob/main/tests/functional/data_providers/test_snowplow_corpus_slate_tracker.py

Reply:
good idea, this is pretty important for monitoring. added